blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -103,45 +106,45 @@ export class Activity extends Component {
         return (<main>
             <article className="activity">
                 { showRunButton &&
-                <RunButton
-                  runnable={pipeline}
-                  latestRun={latestRun}
-                  buttonType="run-only"
-                  onNavigation={onNavigation}
-                  innerButtonClasses="btn-secondary"
-                />
+                    <RunButton
+                      buttonType="run-only"
+                      innerButtonClasses="btn-secondary"
+                      runnable={pipeline}
+                      latestRun={latestRun}
+                      onNavigation={onNavigation}
+                    />
                 }
-                {runs.length > 0 &&
-                <Table className="activity-table u-highlight-rows u-table-lr-indents" headers={headers} disableDefaultPadding>
-                    {
-                        runs.map((run, index) => {
-                            const changeset = run.changeSet;
-                            let latestRecord = {};
-                  
-                            if (changeset && changeset.length > 0) {
-                                latestRecord = new ChangeSetRecord(changeset[changeset.length - 1]);
-                            }
-                            
-                            return (
-                                <Runs {...{
-                                    t,
-                                    locale,
-                                    run,
-                                    pipeline,
-                                    key: index,
-                                    changeset: latestRecord,
-                                }}
-                                />
-                            );
-                        })
-                    }
-                </Table>
+                { runs.length > 0 &&
```

Comment:
Is there any real diff here? It looks like formatting only, but its a dozen odd lines so hard to see - is there some tool doing that as it is a bit annoying as git can't tell it apart from real refactoring.